lld/ELF/Arch/ARM.cpp

```diff
@@ -337,7 +451,7 @@ bool ARM::needsThunk(RelExpr expr, RelType type, const InputFile *file,
   case R_ARM_THM_JUMP24:
     // Source is Thumb, all PLT entries are ARM so interworking is required.
```

Comment:
This comment would be stale and would need updating.

Reply:
Done